.cicd/helpers/docker.sh

```diff
@@ -0,0 +1,17 @@
+function get-envs() {
+    if [[ -f $BUILDKITE_ENV_FILE ]]; then
+        evars=""
+        while read -r var; do
+            evars="$evars --env ${var%%=*}"
+        done < "$BUILDKITE_ENV_FILE"
+    fi
+}
+
+function docker-run() {
```

Comment:
I think using a function in a separate file to call `docker run` is too obtuse. Let's hard-code the `docker run` commands.

Reply:
I agree. This was written when we thought we'd have to run docker run multiple times. We don't anymore so we can eliminate it.